### PR TITLE
Convert sandbox from rabbit to redis OPS-2489

### PR DESCRIPTION
**Pull request by nadeemshahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by nadeemshahzad** [comment]
@edx/devops Please review . thanks

**Review by coryleeio — APPROVED** [review]
This looks good

**Review by jdmulloy — APPROVED** [review]
Looks good to me.

**Review by jdmulloy — CHANGES_REQUESTED** [review]
It looks like the redis role is using a PPA. Do we still need it or should we install from the default Ubuntu reops?

**Comment by nadeemshahzad** [comment]
@jdmulloy I think we need to use chris PPA as default ubuntu repo. has package version 2:3.0.6-1 and using chris PPA we get latest stable version of redis server 4:4.0.5-1chl1~xenial1.
```
redis-server:
  Installed: (none)
  Candidate: 2:3.0.6-1
  Version table:
     2:3.0.6-1 500
        500 http://us-east-1.ec2.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
```
```
  Candidate: 4:4.0.5-1chl1~xenial1
  Version table:
     4:4.0.5-1chl1~xenial1 500
        500 http://ppa.launchpad.net/chris-lea/redis-server/ubuntu xenial/main amd64 Packages

```

**Comment by jdmulloy** [comment]
@nadeemshahzad The newest version currently available from AWS is 3.2.10. Is it possible for us to install this via a PPA in the sandboxes? We probably want an Ansible variable that we can change to update the version in the future.

Thanks

**Comment by nadeemshahzad** [comment]
@jdmulloy Only available version of redis server in PPA for xenial is 4.0.6.
https://launchpad.net/~chris-lea/+archive/ubuntu/redis-server?field.series_filter=xenial

**Review by jdmulloy — CHANGES_REQUESTED** [review]
I agree we should stick with the Chris PPA since it's more up to date. Currentyl we're using 3.2.6 in AWS, so can make Ansible install that version? We should have the version in an Ansible variable so we can change it later.

Thanks

**Comment by jdmulloy** [comment]
@nadeemshahzad Looking at this more and discussing with @feanil we think we should install 3.0 from the Ubuntu repos since it's closer to 3.2 than the 4.0 in the Chris-lea PPA. Thanks

**Review by jdmulloy — APPROVED** [review]

File: playbooks/edx-east/edx_continuous_integration.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     - { role: 'edxapp', celery_worker: True }
     - edxapp
     - testcourses
-    - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
```
Comment (jdmulloy) [inline playbooks/edx-east/edx_continuous_integration.yml]: I think for now we need to keep rabbit in addition to redis because xqueue still uses rabbit.

**Review by jdmulloy — CHANGES_REQUESTED** [review]

**Comment by nadeemshahzad** [comment]
@jdmulloy  I have keep the rabbit role in addition of redis also created base ami for worker,ecommerce, notifier and flower to work with redis. Can you please review these changes? thanks

**Review by jdmulloy — APPROVED** [review]

**Pull request merged**